nima_core/nima_consciousness_core.py

```
@@ -0,0 +1,797 @@
+        """
+        Set a volitional goal.
+        
+        Args:
+            description: Goal description
+            priority: Goal priority (0-1)
+            
+        Returns:
+            Created Goal
+        """
+        if not self.config.enable_volition:
+            raise RuntimeError("Volition is disabled in config")
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)